agent/src/main/java/io/keploy/advice/ksql/RegisterDriverAdvice.java

```diff
@@ -37,16 +36,19 @@ public static void enterMethod(@Advice.Origin Method method, @Advice.Argument(va
                     break;
                 default:
                     System.out.println("Dialect for driver: " + driverClassName + " is not supported yet");
+                    driverClassName = KDriver.class.getName();
+                    break;
             }
+        } else {
```

Comment:
this is wrong implementation At the end of this function the driverClassName should be "io.keploy.ksql.KDriver" in all cases . But in your implementation in the case of "if" condition being true it doesn't happen.

Reply:
ok sir @charankamarapu  i understand ,i will update it accordingly!
,will i do this also in updated one -u can remove the whole method of @Advice.OnMethodExit as it is not used!